curvefs/src/metaserver/storage/rocksdb_storage.cpp

```diff
@@ -42,39 +42,43 @@ const std::string RocksDBOptions::kOrderedColumnFamilyName_ =  // NOLINT
 
 RocksDBOptions::RocksDBOptions(StorageOptions options) {
     // db options
-    RocksDBStorageComparator cmp;
-    dbOptions_.comparator = &cmp;
     dbOptions_.create_if_missing = true;
     dbOptions_.create_missing_column_families = true;
-    dbOptions_.enable_blob_files = true;
     dbOptions_.max_background_flushes = 2;
     dbOptions_.max_background_compactions = 4;
     dbOptions_.bytes_per_sync = 1048576;
-    dbOptions_.compaction_pri = ROCKSDB_NAMESPACE::kMinOverlappingRatio;
-    dbOptions_.prefix_extractor.reset(NewCappedPrefixTransform(3));
 
     // table options
     std::shared_ptr<ROCKSDB_NAMESPACE::Cache> cache =
         NewLRUCache(options.blockCacheCapacity);
     BlockBasedTableOptions tableOptions;
     tableOptions.block_cache = cache;
-    tableOptions.block_size = 16 * 1024;  // 16MB
+    tableOptions.block_size = 16 * 1024;  // 16KB
     tableOptions.cache_index_and_filter_blocks = true;
     tableOptions.pin_l0_filter_and_index_blocks_in_cache = true;
-    tableOptions.filter_policy.reset(NewBloomFilterPolicy(10, true));
-    dbOptions_.table_factory.reset(NewBlockBasedTableFactory(tableOptions));
+    tableOptions.filter_policy.reset(NewBloomFilterPolicy(10, false));
 
     // column failmy options
-    auto unorderedCFOptions = ColumnFamilyOptions();
-    auto orderedCFOptions = ColumnFamilyOptions();
+    comparator_ = std::make_shared<RocksDBStorageComparator>();
+    auto cfOptions = ColumnFamilyOptions();
+    cfOptions.comparator = comparator_.get();
+    cfOptions.enable_blob_files = true;
+    cfOptions.level_compaction_dynamic_level_bytes = true;
+    cfOptions.compaction_pri = ROCKSDB_NAMESPACE::kMinOverlappingRatio;
+    cfOptions.prefix_extractor.reset(NewCappedPrefixTransform(3));
```

Comment:
Does some options here important and add some comments？

Reply:
> Does some options here important and add some comments？

fixed.